liquidator/test/Liquidator.js

```diff
@@ -171,4 +171,17 @@ contract("Liquidator.js", function(accounts) {
     assert.deepStrictEqual(await emp.getLiquidations(sponsor3), []);
     assert.equal((await emp.getCollateral(sponsor3)).rawValue, toWei("175"));
   });
+
+  it.only("set up positions", async function() {
```

Comment:
```suggestion
  it("set up positions", async function() {
```

Reply:
will actually remove this test in it's entirety and move it to a script. I was using it to set up an easy state to execute the liquidator bot from.